test/system/505-networking-pasta.bats

```diff
@@ -145,7 +142,7 @@ function pasta_test_do() {
         local connect="${proto_upper}${ip_ver}:[${addr}]:${one_port}"
         [ "${proto}" = "udp" ] && connect="${connect},shut-null"
 
-        (while sleep ${delay} && ! socat -u "${send}" "${connect}"; do :
+        (while sleep ${delay} && ! socat -u "OPEN:${XFER_FILE}" "${connect}"; do :
```

Comment:
I had a slight preference for the previous version -- sure it's one unnecessary reference, but more symmetric and makes it easier to just change `${send}` instead of having to glance at the `socat` command. I'm fine with this anyway.